### DISCUSSION TITLE
Colored Icon

### DISCUSSION THREAD

**Discussion by Gottarocket** [Arcticons-Team/Arcticons#1265] [Ideas]
I believe this is line based and not just limited to blank lines. So is there any plan to add accent color like orange or one to match wallpaper or background which we can customize. Or something like candy icons. If possible icon animation where color moves around the icon would also be best

**Comment by Donnnno** [comment]
Hi, I've talked about this in other issues and threads before: https://github.com/Donnnno/Arcticons/issues/1223 

The short answer: no, I'm not planning anything like that because it requires too much work.
